roles/setup_lvm_lv/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+debug: False
+
+metal_k8s_lvm_lvs:
+  kubevg:
+    lv01:
+      size: 52G
+    lv02:
+      size: 52G
+    lv03:
+      size: 52G
+    lv04:
+      size: 11G
+    lv05:
+      size: 11G
+    lv06:
+      size: 11G
+    lv07:
+      size: 5G
+    lv08:
+      size: 5G
+
+# You can also customize a specific volume like this
+#
+#      kubevg:
+#        lv01:
+#          size: 10G
+#          fstype: xfs
```

Comment:
Now would be a great time to store FS type (and maybe mount options) as labels on the generated K8s PVs.